### ISSUE TITLE
V3, V7, and V15 Duplicate Problem Entries

### ISSUE THREAD

**Issue by ZaydH** [meelgroup/unigen-benchmarks#4]
In some (maybe all) of the benchmarks in the V3, V7, and V15 folders, there are duplicate problem lines.  An example below is excerpted from [Benchmarks/V7/s27_7_4.cnf](https://github.com/meelgroup/unigen-benchmarks/blob/master/Benchmarks/V7/s27_7_4.cnf?raw=true).

```
**p cnf 24 63**
c
c CNF representation of circuit s27 with a subset ofoutputs randomly xor-ed
c Generated by mMsim
c Total gates: 17 Total inputs 7
c
c
**p cnf 24 63**
c
```

My understanding of the DIMACS specification is that anything other than a single problem line is not in conformance with the DIMACs specification.  I could be wrong here as I am not a super expert on the specification. 

This would be easy to fix if my understanding of the specification is correct.

**Comment by kuldeepmeel** [comment]
Yes, that would be good. Your understanding of DIMACS format is correct. 



—
Kuldeep

**Comment by ZaydH** [comment]
I tried to implement the changes for this, but GitHub is giving me issues as it says [my fork](https://github.com/ZaydH/unigen-benchmarks) is over its LFS limits.  I had hoped the issue would be cleared when August started, but it seems GitHub may track bandwidth on a running 30 day basis - I am not sure.  I also get the LFS quota warning when I try to clone [your repository](https://github.com/meelgroup/unigen-benchmarks).

Since the files are relatively small, would you be open to migrating this off of LFS?  If you think LFS is best here, I would recommend updating the main README that the repository is under LFS.  It may also be prudent how to clone using LFS since I suspect many users who encounter this repository may not be familiar with it (I know I was not).

**Comment by kuldeepmeel** [comment]
If you can migrate this off LFS. Please go ahead and do it. Otherwise, I will look into it (over weekend as this was my first time trying out LFS)

**Comment by ZaydH** [comment]
I do not have a clone of the repository on my machine.  Due to the LFS quota being exhausted, I am not able to create a new clone either.  I think the simpler thing may be to create a new repository and delete this one.
